Return an error if there are two enums with the same name (#3669)
rustwasm/wasm-bindgen: CHANGELOG.md

  enhancing type safety and compatibility.
  [#3647](https://github.com/rustwasm/wasm-bindgen/pull/3647)

> * Stop tolerating name collisions on enums, this would previously just emit one of them.

Reviewer: ```suggestion
* Throw an error on enum name collisions, previously only one enum would be emitted.
```
